source/crud/tests/v2/aggregate-merge.yml

```diff
@@ -0,0 +1,136 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+    - { _id: 3, x: 33 }
+
+minServerVersion: '4.1.12'
+
+collection_name: &collection_name 'test_aggregate_merge'
+
+tests:
+  -
+    description: "Aggregate with $merge"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        arguments: &arguments
+          pipeline: &pipeline
+            - $sort: { x : 1 }
+            - $match: { _id: { $gt: 1 } }
+            - $merge: { into: &output_collection "other_test_collection" }
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+    outcome: &outcome
+      collection:
+        name: *output_collection
+        data:
+          - { _id: 2, x: 22 }
+          - { _id: 3, x: 33 }
+  -
+    description: "Aggregate with $merge and batch size of 0"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        arguments:
+          <<: *arguments
+          batchSize: 0
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            # TODO: check that cursor.batchSize does not exist once
+            # https://jira.mongodb.org/browse/SPEC-1215 has been resolved.
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and majority readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "majority" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "majority" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and local readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "local" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "local" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and available readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "available" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "available" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and linearizable readConcern"
```

Comment:
Why we need a driver test to assert that the server doesn't support linearlizable read concern?  Does it test anything in the driver? Also, we'll just need to update this when the server starts to support it.

Reply:
Will remove.

See: https://github.com/mongodb/specifications/pull/563#discussion_r292032943